plugins/woocommerce/changelog/update-36316

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: tweak
+
+Tweak product link description and display
```

Comment:
Would be good to mention that this is in the new product management experience

Reply:
I've been thinking about this as well.  I don't really love tacking on "in the new product management experience" to every changelog entry because:

1. It's wordy and not easy to quickly find all the relevant changes unless everyone is consistent with their changelog entries.
2. Looking back on the changelog at a later date, using words like "new ..." loses meaning.  Once it becomes the "current" experience we'll have differing entries and if this experience is updated in the future, we'll likely use similar language.

I think going forward it might be good to settle on a prefix for all of these, whether that's "PMEv2: ", "NPME:", "Products v2:", or just "New Product Management Experience:"

But in the meantime I've updated this to `Tweak product link description and display in the new product management experience` until we settle on something more consistent.

Reply:
Good points. There is still the issue of getting everybody to be consistent with it, but I like the idea of a common prefix such as "New Product Editor" (matches how we refer to it in the experimental features screen).